src/components/Carousel.js

```diff
@@ -1,19 +1,85 @@
 import React from 'react';
-
 import './Carousel.scss';
+import Picture from './Picture';
+
+class Carousel extends React.Component {
+  state = {
+    ...this.props,
```

Comment:
```suggestion
    ...this.props,
```
так делать не нужно, я не понимаю что там за свойства